KicadModTree/nodes/specialized/ChamferedPad.py

```diff
@@ -124,17 +129,17 @@ def __getitem__(self, item):
 
     def __setitem__(self, item, value):
         if item in [0, CornerSelection.TOP_LEFT]:
-            self.top_left = value
+            self.top_left = bool(value)
         elif item in [1, CornerSelection.TOP_RIGHT]:
-            self.top_right = value
+            self.top_right = bool(value)
         elif item in [2, CornerSelection.BOTTOM_RIGHT]:
-            self.bottom_right = value
+            self.bottom_right = bool(value)
         elif item in [3, CornerSelection.BOTTOM_LEFT]:
-            self.bottom_left = value
+            self.bottom_left = bool(value)
         else:
             raise IndexError('Index {} is out of range'.format(item))
 
-    def __dict__(self):
+    def to_dict(self):
```

Comment:
reason?

Reply:
given in the commit message. __dict__ does not work in python2. Error message "'dictproxy' object is not callable"

I think __dict__ is reserved for a member that is a dict, not a function returning a dict.
https://docs.python.org/3/library/stdtypes.html#object.__dict__